docker/clients/perplexity_client.py

```
@@ -0,0 +1,127 @@
+        "reasoning_effort": research_effort,
+        "return_citations": True,
+        "return_related_questions": False,
+        "search_domain_filter": [],  # Search all domains
+        "temperature": 0.2,  # Lower temperature for factual research
+    }
+
+    try:
+        logger.info(
+            f"Fetching Perplexity deep research (effort: {research_effort}): {query}"
+        )
+        response = requests.post(url, headers=headers, data=json.dumps(payload))
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)